statics/js/components/workflow.js

```diff
@@ -0,0 +1,141 @@
+Vue.component('item', {
+  name: 'workflow-item',
+
+  props: [
+    'Name',
+    'Description',
+    'Type',
+  ],
+
+  inject: [
+    'formData'
+  ],
+
+  template: `
+  <div class="form-group">\
+    <label :for="name">{{Description}}</label>
+    <textarea v-if="Type == 'string'" :id="name" v-model="formData[Name]"></textarea>
+    <input v-else-if="Type == 'date'" type="date" :id="name" v-model="formData[Name]" class="form-control input-sm">
+    <input v-else-if="Type == 'integer'" type="number" :id="name" v-model="formData[Name]" class="form-control input-sm">
+    <node-selector v-else-if="Type == 'node'" :id="name" v-model="formData[Name]"></node-selector>
+    <template v-else-if="Type == 'boolean'" v-for="option in ['yes','no']">
+      <input  type="radio" :id="name + option" v-model="formData[Name]" :value="option" class="form-control input-sm">
+      <label :for="name + option">{{option}}</label>
+    </template>
+    <template v-else-if="Type == 'choice'" v-for="option in ['yes','no','dont know']">
+    <label class="radio-inline">\
+      <input  type="radio" :id="name + option" v-model="formData[Name]" :value="option"  class="form-control input-sm">
+      <label :for="name + option">{{option}}</label>
+    </label>
+    </template>
+    <template v-else-if="Type == 'group'">
+      <item v-for="i in item" v-bind="i" :key="i.name"></item>
+    </template>
+  </div>`
+
+})
+
+Vue.component('workflow-params', {
+  props: [
+      'workflow'
+  ],
+
+  data() {
+  	return {
+      formData: {
+      }
+    }
+  },
+
+  provide() {
+  	return {
+        formData: this.formData
+    }
+  },
+
+  inject: [
+    'result'
+  ],
+
+  template: `
+    <form v-if="workflow.Parameters" @submit.prevent="submit">
+      <h1>{{workflow.Name}}</h1>
+      <item v-for="item in workflow.Parameters" v-bind="item" :key="item.Name"></item>
+      <button type="submit">Submit</button>
```

Comment:
The button doesn't have the correct style

Reply:
Fixed